doc/getting-started-guide/index.rst

```diff
@@ -13,3 +13,4 @@ It is designed as an entry point for new users, and it provided an introduction
    installing
    quick-overview
    faq
+   help-diagram
```

Comment:
Shall we move this to the top-level index.rst instead?

Reply:
Moved, but happy to put back within "Getting Started" if that's the overall preferred spot.